Changelog.md

```diff
@@ -5,6 +5,7 @@ Language Features:
 
 
 Compiler Features:
+ * Code Generator: Use ``calldatacopy`` past input to zero out memory instead of ``codecopy``.
```

Comment:
```suggestion
 * Code Generator: Use ``calldatacopy`` instead of ``codecopy`` to zero out memory past input.
```
Maybe it's easier to understand like that?